enterprisemediawiki/meza
# Enable time sync

This addition installs packages required in order to sync a VM with a time service. These steps will ensure the VM time is correct for each of the log entries, as added through cmd_tee(). I chose to put this in-line in install.sh since the steps were minimal and if these were instead in another script, the log timestamps might be offset from the subsequent entries. Closes #118.

### Testing
* SSH into a VM and use `date` to see the current timestamp
* Use `date -s "31 OCT 2015 18:00:00"` to set the server to an incorrect timestamp
* Use `date` to verify your change
* `curl -LO https://raw.githubusercontent.com/enterprisemediawiki/meza/time-sync/scripts/install.sh`
* `sudo bash install.sh`
* Use the `time-sync` branch
* After installation is complete, use `date` to verify the date has been updated from the time service
* `service ntpd status` to verify the service is running
